src/Field/MineSquare.java

```diff
@@ -1,8 +1,8 @@
 package Field;
 
-public class MineSquare implements Square {
+public class MineSquare extends Square {
 
-    private String symbol = "*";
+    private String stringValue = "*";
```

Comment:
Can you think of a better name of this variable?

Reply:
mineSymbol?

Reply:
Possibly. That said though, does this need to be a private field? Is it ever modified?

Reply:
symbol, icon, or token maybe. I know you renamed it from "symbol" I'm not sure what the logic here was. also agree with Nik on it needing to be a field.

I would say mineSymbol doesn't make sense, because we know it is a mine already because it's in the mineSquare class. However, "mineSymbol" is better than stringValue. This is because all strings either have a value or are null

Reply:
The easiest things to name are the things that don't need naming at all 😂

Reply:
It's the minor details that seem to be the hardest to name. It is never modified which is why I thought it would be private?

Reply:
You can just make it a constant at the top of your file. Doesn't have to be private.